### PR TITLE
Add linear elastic material model to documentation

### PR DESCRIPTION
**Pull request by jgiven100**
# Describe the PR
Starting the `linear elastic` online documentation. 

### Inputs:
* youngs modulus
* poisson ratio
* density

### Defined values:
* bulk, shear, and constrained moduli
* shear and pressure wave velocity

### Reference(s):
* Al-Kafaji, I. K. J. (2013)

# Related Issues/PRs
Features added to code here: https://github.com/cb-geo/mpm/pull/705
Documentation issue described here: https://github.com/cb-geo/mpm-doc/issues/59

# Additional context
Combined effort with @jgiven100 and @cgeudeker

### CODE REVIEW

File: theory/material/linear-elastic.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Wave Velocities
+
+The shear and pressue wave velocities ($v_s$ and $v_p$, respectively) are defined [1] as follows,
+
+$$ v_s = \sqrt{\frac{M}{\rho}}, $$
+
+$$ v_p = \sqrt{\frac{G}{\rho}}, $$
+
+where $\rho$ is the density.
+
```
Comment (kks32) [inline theory/material/linear-elastic.md:26]: Could you please add that these information are automatically computed and is available as a JSON material property and can be accessed as: `property["pwave_velocity"]` and `property["swave_velocity"]`. I think that would be useful.

**Review by kks32 — CHANGES_REQUESTED** [review]
Thanks @jgiven100 looks great.

File: theory/material/linear-elastic.md (hunk 1 above)
Comment (jgiven100) [reply theory/material/linear-elastic.md:26]: Automatic computation and storage as JSON material properties added in newest commit

**Review by kks32 — APPROVED** [review]

**Comment by bodhinandach** [comment]
Thanks for this @jgiven100

**Pull request merged**